book/README.md

```diff
@@ -1,53 +1,73 @@
-# TLG Catalog <a href='https://insightsengineering.github.io/tlg-catalog/'><img src="assets/img/logo.png" align="right" height="139" style="max-width: 100%; max-height: 139px;"/></a>
+# <a href='https://insightsengineering.github.io/tlg-catalog/'><img src="assets/img/logo.png" align="right" height="139" style="max-width: 100%; max-height: 139px;"/></a>
 
-The catalog for (T)ables, (L)istings, and (G)raphs for clinical trials generated using NEST packages.
+The TLG catalog is a catalog of **T**ables, **L**istings, and **G**raphs for clinical trials generated using NEST packages.
 
 This repository provides a comprehensive collection of clinical trials outputs generated using the R language.
 The target audience is the clinical trials community, including statisticians, data scientists, and other professionals interested in applying R to clinical trials data.
 
 ## Usage
 
-Each output is generated in a separate article and this usually consists of:
+Each TLG is represented on a separate article page, typically including the following sections:
 
-* Data setup using synthetic data.
-* The output itself (many variants if applicable).
-* The output embedded in an interactive application.
-* Reproducibility information.
+- Setup and pre-processing of synthetic data.
+- Steps to produce the TLG.
+- The output TLG generated by the given code (including any available variants).
+- An interactive application that can alternatively be used to produce and interact with the TLG.
+- Reproducibility information.
 
-See the full list of available TLGs in [Index page](tlg-index.qmd).
+See the full list of available TLGs on the [Index page](tlg-index.qmd).
 
-The source code of each article can be inspected by clicking on the "Source Code" button. Use copy-paste to get the code and run it in your environment.
+### Interacting with Catalog R Code
 
-![artice code copy](assets/img/article-code-copy.gif)
+::: columns
+::: {.column style="vertical-align: middle; width: 30%; padding-right: 5%"}
+The full source code of each article can be viewed by clicking on the "Source Code" button at the top of the page and copied using the "Copy to Clipboard" button.
+:::
```

Comment:
This does not render in my preview -> I think there is an extra `:::`?

Reply:
This section renders as expected for me. Are you using `quarto 1.3`? The double `:::` atthe top and bottom are used to define the overall column section.

![Screenshot 2024-02-28 102314](https://github.com/insightsengineering/tlg-catalog/assets/59304861/ae0a06c8-d988-468f-853e-673cf8494f42)

Reply:
I get this: 

<img width="1145" alt="Screenshot 2024-02-29 094128" src="https://github.com/insightsengineering/tlg-catalog/assets/11279768/8f4d8f33-32ef-4ec9-9436-ca223c596dd9">

Reply:
It would be good to keep a base markdown (which is also supported in Quarto) for GitHub Preview also -> https://github.com/insightsengineering/tlg-catalog/tree/224_book_readme%40main/book